docker/openEuler/compile/Dockerfile

```diff
@@ -0,0 +1,113 @@
+FROM opencurve/curve-base-openeuler:latest
+
+ENV GITHUB_PROXY=https://ghproxy.com/
+ENV ETCD_VERSION=v3.4.0 \
+    DOWNLOAD_ETCD_URL=${GITHUB_PROXY}https://github.com/etcd-io/etcd/releases/download \
+    GOPROXY=https://goproxy.io,direct \
+    PROTOC_VERSION=21.8 \
+    GO_VERSION=1.19.6 \
+    PROTOC_GEN_GO_VERSION=v1.28 \
+    PROTOC_GEN_GO_GRPC_VERSION=v1.2
+COPY uuid/* /root/curve_compile_openEuler/uuid/
+COPY curve_base_openEuler.tar.gz /root/curve_compile_openEuler/curve_base_openEuler.tar.gz
+COPY libfiu-master.zip /root/curve_compile_openEuler/
```

Comment:
You can use staged compilation of some dependent libraries, like this:
Compiled in [curve-tgt](https://github.com/opencurve/curve/blob/master/docker/debian9/curve-tgt/Dockerfile), in [curvebs](https://github.com/opencurve/curve/blob/master/docker/debian9/Dockerfile) uses COPY to copy the compiled image from the previous image, and you can use `git clone --branch xxx` or `wget xxx` to get the source code.